packages/pigeon/lib/dart_generator.dart

```diff
@@ -587,6 +588,11 @@ pigeonMap['${field.name}'] != null
       });
     }
 
+    void writeToString() {
+      indent.writeln('@override');
+      indent.writeln('String toString() => jsonEncode(encode());');
```

Comment:
Sorry, I don't think we should do this.  It is leaking information about the wireformat to the public API.  Remember we want to be able to completely isolate the wireformat from users so that we can switch it out without breaking people.

Reply:
If you still want a `toString` method it shouldn't rely on `encode()` and it it probably shouldn't output a format that is trivially parsed.